doc/reference/reference_lua/box_snapshot.rst

```diff
@@ -74,3 +76,22 @@
     in automation: a signal provides no way to find out whether the snapshot
     was taken successfully or not.
 
+    **Vinyl**
+
+    In vinyl, inserted data is stacked in memory until the limit, set in
+    :ref:`vinyl_memory <cfg_storage-vinyl_memory>` parameter, is reached. Then
+    vinyl automatically dumps it to the disc. ``box.snapshot()`` forces
+    this dump in order to have the ability to recover from this checkpoint.
+    The snapshot files are stored in :samp:`{space_id}/{index_id}/*.run`.
+    Thus, strictly all the data that was written at the time of LSN of the
+    checkpoint is in the `*.run` files on the disk, and all operations that happened
```

Comment:
Please mark all file names in this commit with double quotes:

```
 ``*.run`` but not `*.run`
```

Single quotes will be rendered in italics, which is harder to read than the monotype font that we get with double quotes.

Reply:
fixed in 36c09b2